record-and-playback/core/lib/recordandplayback/edl/audio.rb

```diff
@@ -41,6 +41,25 @@ def self.dump(edl)
         end
       end
 
+      def self.mixer(inputs, output_basename)
+        BigBlueButton.logger.debug "Mixing audio files"
+
+        ffmpeg_cmd = [*FFMPEG]
+        inputs.each do |input|
+          ffmpeg_cmd += ['-i', input]
+        end
+        ffmpeg_cmd += ['-filter_complex', "amix"]
```

Comment:
We should explicitly set the number of inputs to the amix filter here, in case a number of inputs ≠ 2 is provided.
```suggestion
        ffmpeg_cmd += ['-filter_complex', "amix=inputs=#{inputs.length}"]
```